tests_integration/tools/test_relevant_news_analysis.py

```
@@ -0,0 +1,98 @@
+
+
+# TODO test cache and get_certified_relevant_news_since_cached
+def test_get_certified_relevant_news_since_cached() -> None:
+    cache = RelevantNewsResponseCache(sqlalchemy_db_url="sqlite:///:memory:")
+
+    question = (
+        "Will the price of Bitcoin be higher than $100,000 by the end of the year?"
+    )
+    days_ago = 5
+    assert (
+        cache.find(question=question, days_ago=days_ago) == None
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)